yaml/schema/mod.ts

```diff
@@ -19,7 +18,7 @@ export {
 };
 
 export function replaceSchemaNameWithSchemaClass(
-  options?: { schema?: string | Schema },
+  options?: { schema?: string | unknown },
```

Comment:
Ditto elsewhere where this type is used.
```suggestion
  options?: { schema?: "core" | "default" | "failsafe" | "json" | "extended" | unknown },
```